php/commands/site.php

```diff
@@ -776,30 +743,63 @@ public function merge_from_dev($args, $assoc_args) {
      }
 
    /**
-   * Delete a MultiDev environment
-   *
-   * ## OPTIONS
-   *
-   * [--site=<site>]
-   * : Site to use
-   *
-   * [--env=<env>]
-   * : name of environment to delete
-   *
-   * @subcommand delete-env
-   */
-   public function delete_env($args, $assoc_args) {
-     $site = SiteFactory::instance(Input::sitename($assoc_args));
-     $site->environmentsCollection->fetch();
-     $multidev_envs = array_diff($site->environmentsCollection->ids(), array('dev', 'test', 'live'));
-     $env = Input::env($assoc_args, 'env', "Environment to delete", $multidev_envs);
+    * Delete a multidev environment's branch
```

Comment:
Just a description thing. At this point (shouldn't) be a multidev environment anymore, so it's just a git branch,